From: Takashi Iwai <tiwai@suse.de>
Subject: Fix kABI breakage due to sound/timer.h inclusion
Patch-mainline: Never, SLE15 kABI
References: bsc#1051510

The patch patches.drivers/ALSA-pcm-Fix-UAF-at-PCM-release-via-PCM-timer-access
adds the inclusion of <sound/timer.h>, and it broke kaBI like:
  sound/core/pcm.c:756: warning: snd_pcm_new_stream: modversion changed because of changes in struct snd_timer (became defined)

Wrap the new inclusion with ifdef.

Signed-off-by: Takashi Iwai <tiwai@suse.de>

---
 sound/core/pcm.c |    2 ++
 1 file changed, 2 insertions(+)

--- a/sound/core/pcm.c
+++ b/sound/core/pcm.c
@@ -28,7 +28,9 @@
 #include <sound/core.h>
 #include <sound/minors.h>
 #include <sound/pcm.h>
+#ifndef __GENKSYMS__
 #include <sound/timer.h>
+#endif
 #include <sound/control.h>
 #include <sound/info.h>
 
